C++/C++.sublime-syntax

```diff
@@ -209,6 +209,13 @@ contexts:
     - include: preprocessor-statements
     - include: scope:source.c#label
     - include: expressions
+    
+  statements-enum:
+    - include: preprocessor-statements
+    - include: scope:source.c#label
+    - match: '{{identifier}}'
+      scope: entity.name.enum.c++.constants
```

Comment:
This scope should be `entity.name.constant.c++` since that is what is indexed. The scope `entity.name.enum` is used for the enum type name. A note for future changes, the syntax scope (`.c++`) should always come last.